fiware-region-sanity-tests/commons/fiware_cloud_test_case.py

```diff
@@ -320,7 +320,7 @@ def reset_world_ports(cls, world, suite=False):
         for port_id in list(world['ports']):
             try:
                 port_data = cls.neutron_operations.show_port(port_id)
-                if 'network:router_interface' == port_data['device_owner']:
+                if 'network:router_interface' in port_data['device_owner']:
```

Comment:
I was testing the new feature in our Jenkins at the end of the friday and I saw this problem. This new code for E2E testing is not merged to master yet. Fiware-Helath's Jenkins is not running this code yet. I will continue today testing it in all regions.